frame/staking/src/tests.rs

```diff
@@ -4710,17 +4711,19 @@ fn on_initialize_weight_is_correct() {
 
 #[test]
 fn payout_creates_controller() {
-	ExtBuilder::default().has_stakers(false).build_and_execute(|| {
```

Comment:
style note, i much prefer this original style because you dont need to introduce so much whitespace to all the lines below

Reply:
dont change this for me though